Makefile.defs

```diff
@@ -143,7 +143,9 @@ endif
 # Makefiles do some tricky stuff to differentiate HOST/TARGET
 BASE_CFLAGS:=-O2 -ffast-math -pipe -fomit-frame-pointer
 # Use this for debugging:
-#BASE_CFLAGS:=-O0 -g
+ifdef KODEBUG
+    BASE_CFLAGS:=-Og -g
```

Comment:
Wait, is there a reason this line was different? Shouldn't it just be the same as regular (`-ffast-math` etc) but with `0g -g`?

Reply:
Dunno, may be updating it was just forgotten when updating the main one (check commit history maybe ?)
OK with `KODEBUG` , it sounds nice :)